docs/source/sycamore/tutorials/intermediate_etl_tutorial.md

```diff
@@ -0,0 +1,7 @@
+# Intermediate ETL with Sycamore and Pinecone
+
+This tutorial is meant to show how to create an ETL pipeline with Sycamore to load a Pinecone vector database. It walks through an intermediate ETL flow: partitioning, extraction, cleaning, chunking, embedding, and loading. You will need an [Aryn Partitioning Service API key](https://www.aryn.ai/get-started), [OpenAI API key](https://platform.openai.com/signup) (for LLM-powered data enrichment and creating vector embeddings), and a [Pinecone API key](https://app.pinecone.io/?sessionType=signup) (for creating and using a vector index). At the time of writing, there are free trial or free tier options for all of these services.
```

Comment:
Curious, what do we mean by "intermediate" here? Is this a technical term or are we talking about it like beginner, intermediate, advanced?

Reply:
I mean it in the latter sense. Is that not clear?

Reply:
I didn't think of it that way at first, mostly because we don't have any "beginner" or "advanced" tutorials to accompany (or we haven't used that terminology yet). I don't feel strongly about it, but I'd err on the side of not using that term because my first thought was this is some sort of "intermediate step" in an ETL flow.

Reply:
Ack